### DISCUSSION TITLE
Problem installing

### DISCUSSION THREAD

**Discussion by mrsbpier** [ant-media/Ant-Media-Server#5832] [Q&A]
Hi, I tried to install Ant Media Server on my hosting server (linux) with the terminal on c-panel but I receive this error
<img width="589" alt="Screenshot 2023-11-20 alle 19 21 10" src="https://github.com/ant-media/Ant-Media-Server/assets/84006973/bf203011-027b-4df4-97c7-a0e267c7d8ee">

**Comment by muratugureminoglu** [comment]
Hi @mrsbpier 

You are encountering a permission error. Could you share with me how you run the install script?

Normally you should run it as follows (with `sudo` command).
```
sudo ./install_ant-media-server.sh -i <ANT_MEDIA_SERVER_ZIP_FILE>
```
